### DISCUSSION TITLE
Can I use my Android Tablet?

### DISCUSSION THREAD

**Discussion by jottbecr** [xournalpp/xournalpp#5232] [Q&A]
Hi, 

can I somehow make my Android Tablet a Graphical Tablet. In other words, can I use my Android Tablet as an input device of my Linux Machine with xournal++ like e.g. a Wacom One?
And if so, what do I have to install in addition to Xournal++ on my Ubuntu machine and what on my tablet?
Sorry if this question is already answered, I couldn't find it in the forum.

Thank you in advance
Jürgen

**Comment by rolandlo** [comment]
You can try with [Weylus](https://github.com/H-M-H/Weylus#weylus). In case you do, let us know how it worked.
